src/routes/(admin)/users/+page.svelte

```
@@ -24,28 +27,104 @@
+
 	// handle sorting
 	function handleSort(columnId: string) {
 		const url = new URL(page.url);
 		const newSortOrder = sortBy === columnId && sortOrder === 'asc' ? 'desc' : 'asc';
 
 		url.searchParams.set('sortBy', columnId);
 		url.searchParams.set('sortOrder', newSortOrder);
-		url.searchParams.set('page', '1'); // reset to first page when sorting
+		url.searchParams.set('page', '1');
 
 		goto(url.toString(), { replaceState: false });
```

ESLint

[error] Unexpected goto() call without resolve().

(svelte/no-navigation-without-resolve)